.github/workflows/install.sh

```diff
@@ -29,3 +29,9 @@ elif [ "${flavor}" = "docs" ]; then
   poetry install --verbose --no-interaction --with=docs --extras=interpolation
 
 fi
+
+# FIXME: Remove this again.
+# Fix dependency woes about percy: AttributeError: module 'percy' has no attribute 'Runner'.
+# https://github.com/earthobservations/wetterdienst/pull/1017#issuecomment-1741240635
+# pytest tests/ui/explorer/test_explorer.py -k test_app_layout
+pip install "percy>=2" --force
```

Comment:
Q: What's that?
A: See my report at https://github.com/earthobservations/wetterdienst/pull/1017#issuecomment-1741240635 ff.